keyboards/1upkeyboards/super16/keymaps/15game/keymap.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2020 Sam Reinehr
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+enum my_keycodes {
+    K00 = SAFE_RANGE,
+    K01,
+    K02,
+    K03,
+    K04,
+    K05,
+    K06,
+    K07,
+    K08,
+    K09,
+    K10,
+    K11,
+    K12,
+    K13,
+    K14,
+    K15,
+};
+/* just a simple way to give each key a unique code */
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {[0] = LAYOUT_ortho_4x4(/* Base */
+                                                                                     K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K10, K11, K12, K13, K14, K15)};
```

Comment:
```suggestion
// clang-format off
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [0] = LAYOUT_ortho_4x4(/* Base */
        K00, K01, K02, K03,
        K04, K05, K06, K07,
        K08, K09, K10, K11,
        K12, K13, K14, K15
    )
};
// clang-format on
```

Reply:
I've made a few more formatting changes to make the array structure present in all the maps